### DISCUSSION TITLE
Similar extension for Snap by Berkeley

### DISCUSSION THREAD

**Discussion by obdow** [ScratchAddons/ScratchAddons#7087] [Ideas]
In my opinion snap is much better in almost every way with higher potential for amazing projects, but it’s been neglected and an add on like this would be great to polish it and make it as streamlined as Scratch. Had anyone thought about making something like it?

**Comment by pufferfish101007** [comment]
Snap_!_ is generally a bit more open to open-source contributions from what I've seen, so if people wanted to add things like what scratch addons does, they could probably get it upstreamed into Snap_!_ itself. Hence, probably not much point in making an extension.

**Comment by BroJac5246** [comment]
I'm not sure this is the right place to ask because most people here are focused on Scratch and are aware of but perhaps not incredibly familiar with Snap!. I like the idea but it might make more sense to seek out a dedicated Snap! community that would be interested in building such an extension.
